usagov/vote-gov-drupal
# Bug/vote 2077/state selector axe error

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2077

## Description

Remove listbox role from the state selector. It is not the correct aria role, and caused some axe errors.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Visit the registration landing page and run axe tools (in browser or cypress)
2. Confirm there are no axe errors produced by the state selector.

## Checklist for the Developer




## Checklist for the Peer Reviewers
